.github/workflows/CI.yml

```diff
@@ -2,6 +2,9 @@ name: 🧪 Integration Tests
 
 on: [push, pull_request, workflow_dispatch]
 
+env:
+  DB_URL: postgresql://postgres:postgres@postgres:5432/chowda-test
+
```

Comment:
i think DB_URL will get overridden when the tests are run, i.e.
1. CHOWDA_ENV gets set to "test" automatically whenever running pytest
2. the .env.test is loaded, which contains the DB_URL

at least that's how it's supposed to work. Is it erroring without this explicit env var setting?

Reply:
Yes, and:

- `.env.test` contains test values for testing from a local machine (db: localhost)
- For CI, we override that with the url for the postgres service
  - Because we removed `override=True`, we can just set this variable, instead of:
    - Create `.env.ci` with `DB_URL`
    - Set `CHOWDA_ENV=CI`
    - Add a switch case in `conftest.py` for CI

Reply:
## Bug
When I fixed the CI file, the tests had this bug: https://github.com/WGBH-MLA/chowda/actions/runs/5479062625/jobs/9980381904
> E   sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "chowda-development" does not exist

## Import order
Fixing the import order in `conftest.py` then used the correct dotfile: https://github.com/WGBH-MLA/chowda/actions/runs/5479131791/jobs/9980556927

> E   sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "chowda-test" does not exist

## Override
Setting `env: DB_URL` successfully connected to the db and rand the tests: https://github.com/WGBH-MLA/chowda/actions/runs/5479229000/jobs/9980783510